cicd/gitops/configuring-sso-on-argo-cd-using-dex.adoc

```diff
@@ -8,18 +8,13 @@ toc::[]
 
 After the {gitops-title} Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, cluster administrators can use Argo CD to configure Single Sign-On (SSO).
 
-[IMPORTANT]
```

Comment:
@svghadi As this is still supported until v1.10, should we be just changing the version here instead of removing this warning as a whole or it will be a release notes item?

Reply:
Yes. Makes sense to keep it.

Reply:
@masauso-lungu - Instead of completely removing `[IMPORTANT]` message, can we convert it to a `[note]` or `[warning]` having something like
> The `.spec.dex` parameter in the ArgoCD CR is no longer support from {gitops-title} v1.10 onward. Consider using the `.spec.sso` parameter instead.

Basically to inform users who are looking for older configurations.